internal/database/repository/partitions.go

```diff
@@ -88,3 +126,25 @@ func (s *PostgresRepository) GetAllPartitions(ctx context.Context) ([]*dao.Parti
 	}
 	return partitions, nil
 }
+
+func (s *PostgresRepository) DeletePartitions(ctx context.Context, partitions []*dao.PartitionInfo) error {
```

Comment:
Rename to `DeleteActivePartitionsNotInPartitions` or something similar. The name is misleading :relaxed:

Reply:
Updated the name, but still not sure about the wording 😐. Let me know how it sounds.